WebApp/Content/styles/views/dashboard.less

```diff
@@ -557,7 +557,25 @@ p{
         overflow-y: auto;
         max-height: none;
     }
- }
+ .dashboard_contextpanel_list_container {
+        margin-left: 30px;
+    }
+ .dashboard_contextpanel_select_filter_button, 
+ .dashboard_contextpanel_select_filter_alert_button {
+        left: 20px;
+    }
+ .graph-header {
```

Comment:
unless those are not library classes, let stick with the underscore naming convention